### PR TITLE
Ji 89 today's serving has ended implementation with unit test

### PR DESCRIPTION
**Pull request by OmarSemou**

### CODE REVIEW

**Comment by OmarSemou** [comment]
flutter test fails because the default basic flutter is failing but I figure that we will not be using it in the future, otherwise I will work on fixing the test

File: lib/services/service_time_checker.dart
Comment (allanzigge) [inline lib/services/service_time_checker.dart:1]: Why 20.59, not 21.00?

**Review by allanzigge — CHANGES_REQUESTED** [review]
Approved. 

As agreed we will merge next monday, therefore I put "request change" untill then :)

File: lib/services/service_time_checker.dart
Comment (OmarSemou) [reply lib/services/service_time_checker.dart:1]: Because the story states that the message should display from 21 therefore after 20:59

**Review by allanzigge — APPROVED** [review]

**Pull request merged**